### PR TITLE
Add user-friendly error messages for known issues.

### LINKED ISSUE #4074: hyperv: no External vswitch found. A valid vswitch must be available for this command to run
minikube start --vm-driver=hyperv -v 10

PS C:\WINDOWS\system32> minikube start --vm-driver=hyperv -v 10
o   minikube v1.0.0 on windows (amd64)
$   Downloading Kubernetes v1.14.0 images in the background ...
>   Creating hyperv VM (CPUs=2, Memory=2048MB, Disk=20000MB) ...
@   Downloading Minikube ISO ...
 142.80 MB / 142.88 MB [============================================]  99.94% 0s
 142.88 MB / 142.88 MB [============================================] 100.00% 0sCreating CA: C:\Users\somuser\.minikube\certs\ca.pem
Creating client certificate: C:\Users\someuser\.minikube\certs\cert.pem
[executing ==>] : C:\WINDOWS\System32\WindowsPowerShell\v1.0\powershell.exe -NoProfile -NonInteractive @(Get-Module -ListAvailable hyper-v).Name | Get-Unique
[stdout =====>] : Hyper-V

[stderr =====>] :
[executing ==>] : C:\WINDOWS\System32\WindowsPowerShell\v1.0\powershell.exe -NoProfile -NonInteractive @([Security.Principal.WindowsPrincipal][Security.Principal.WindowsIdentity]::GetCurrent()).IsInRole("S-1-5-32-578")
[stdout =====>] : False

[stderr =====>] :
[executing ==>] : C:\WINDOWS\System32\WindowsPowerShell\v1.0\powershell.exe -NoProfile -NonInteractive @([Security.Principal.WindowsPrincipal][Security.Principal.WindowsIdentity]::GetCurrent()).IsInRole([Security.Principal.WindowsBuiltInRole] "Administrator")
[stdout =====>] : True

[stderr =====>] :
[executing ==>] : C:\WINDOWS\System32\WindowsPowerShell\v1.0\powershell.exe -NoProfile -NonInteractive [Console]::OutputEncoding = [Text.Encoding]::UTF8; (Hyper-V\Get-VMSwitch -SwitchType External).Name
[stdout =====>] :
[stderr =====>] :

!   Unable to start VM: create: precreate: no External vswitch found. A valid vswitch must be available for this command to run. Check https://docs.docker.com/machine/drivers/hyper-v/

*   Sorry that minikube crashed. If this was unexpected, we would love to hear from you:
-   https://github.com/kubernetes/minikube/issues/new

Windows 10

### PR DESCRIPTION
**Pull request by sharifelgamal**
Fixes #4074

### CODE REVIEW

File: pkg/minikube/problem/err_map.go [hunk 1]
```diff
@@ -61,6 +61,11 @@ var vmProblems = map[string]match{
 		Advice: "Disable Hyper-V when you want to run VirtualBox to boot the VM",
 		Issues: []int{4051},
 	},
+	"HYPERV_NO_VSWITCH": {
+		Regexp: re(`no External vswitch found. A valid vswitch must be available for this command to run.`),
+		Advice: "Hyper-V requires an external network switch to function properly.",
```
Comment (tstromberg) [inline pkg/minikube/problem/err_map.go]: The advice here is intended to be worded as something the user should perform to work around this. Like perhaps: "Configure an external network switch using x-y command"

File: pkg/minikube/problem/err_map.go [hunk 2]
```diff
@@ -61,6 +61,11 @@ var vmProblems = map[string]match{
 		Advice: "Disable Hyper-V when you want to run VirtualBox to boot the VM",
 		Issues: []int{4051},
 	},
+	"HYPERV_NO_VSWITCH": {
+		Regexp: re(`no External vswitch found. A valid vswitch must be available for this command to run.`),
```
Comment (tstromberg) [inline pkg/minikube/problem/err_map.go]: Please add a test.

Comment (sharifelgamal) [reply pkg/minikube/problem/err_map.go]: Do we have tests for this file anywhere?

File: pkg/minikube/problem/err_map.go (hunk 1 above)
Comment (sharifelgamal) [reply pkg/minikube/problem/err_map.go]: Done.

File: pkg/minikube/problem/err_map.go (hunk 2 above)
Comment (tstromberg) [reply pkg/minikube/problem/err_map.go]: well.. I guess not! I swore I wrote some... hum.

**Pull request merged**